content/blog/2019/08/2019-08-27-role-strategy-performance.adoc

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "Performance Improvements to Role Strategy Plugin"
+tags:
+- plugins
+- security
+- performance
+- gsoc
+- gsoc2019
+author: abhyudayasharma
+---
+
+The task for my Google Summer of Code program was to improve the performance
+of the Role Strategy Plugin. We started by creating a
+link:/blog/2019/06/21/performance-testing-jenkins/[framework]
+for running micro-benchmarks on Jenkins Plugins. Benchmarks using the framework
+were added to the plugin:role-strategy[Role Strategy Plugin] find performance
+critical parts of the plugins and to measure the improvements of a change.
+This blog post summarizes the changes that were made and performance improvements
+measured.
+
```

Comment:
It would be also nice to have a paragraph which summarizes the initial situation (similar to what you've done in your presentation) and the prior work (commits by Deepansh)

Reply:
@oleg-nenashev Does it look better with bc25cb4?